File: lec_33_file_handling/multiply.cpp
#include <bits/stdc++.h>
using namespace std;

int main(void){

  ifstream file("input.txt");
  if(!file){
    cout << "Error in opening file" << endl;
    return -1;
  }

  int a, b;
  file >> a >> b;
  file.close();

  ofstream fw("output.txt");
  fw<< "The product of " << a << " and " << b << " is " << a * b << " - written from the C++ program"<< endl;

return 0;
}

File: lec_33_file_handling/Multiply.java
import java.io.BufferedReader;
import java.io.FileNotFoundException;
import java.io.FileReader;
import java.io.FileWriter;
import java.io.IOException;

public class Multiply{
  public static void main(String[] args){
    try{
      BufferedReader fr = new BufferedReader(new FileReader("input.txt"));
      String line = fr.readLine();
      int a = Integer.parseInt(line);
      line = fr.readLine();
      int b = Integer.parseInt(line);
      fr.close();

      try {
        FileWriter fw = new FileWriter("output.txt" , true);
        fw.write("The product of " + a + " and " + b + " is " + a * b + " - written from the Java program");
        fw.close();
      }
      catch(FileNotFoundException err){
        err.printStackTrace();
      }

    }
    catch(IOException err){
      err.printStackTrace();
    }
  }
}